src/main/java/uk/ac/ebi/embl/flatfile/writer/embl/COWriter.java

```diff
@@ -41,7 +41,7 @@ public boolean write(Writer writer) throws IOException {
       } else {
         firstContig = false;
       }
-      FeatureLocationWriter.renderLocation(block, contig, false, false);
+      FeatureLocationWriter.renderLocation(block, contig);
```

Comment:
We never want to write partiality for CO lines. The old code prevented this completely. Now we will write partiality if the location has it.